test/functional/legacy/077_mf_hash_grow_spec.lua

```diff
@@ -0,0 +1,45 @@
+-- Inserts 2 million lines with consecutive integers starting from 1
+-- (essentially, the output of GNU's seq 1 2000000), writes them to Xtest
+-- and calculates its cksum.
+-- We need 2 million lines to trigger a call to mf_hash_grow().  If it would mess
+-- up the lines the checksum would differ.
+-- cksum is part of POSIX and so should be available on most Unixes.
+-- If it isn't available then the test will be skipped.
+
+local helpers = require('test.functional.helpers')
+local feed, insert, source = helpers.feed, helpers.insert, helpers.source
+local clear, execute, expect = helpers.clear, helpers.execute, helpers.expect
+
+describe('mf_hash_grow()', function()
+  setup(clear)
+
+  if os.execute('which cksum &> /dev/null') == 0 then
```

Comment:
forgot to mention, this is a bash feature not common on BSD. Can you change this to `2>&1`, see 3562f686b0a8ec0051ceaa12110be354c34fc0a0 for example.